wolfssl/wolfcrypt/port/atmel/atmel.h

```diff
@@ -108,21 +121,55 @@ int  atmel_get_rev_info(word32* revision);
 void atmel_show_rev_info(void);
 
 WOLFSSL_API int wolfCrypt_ATECC_SetConfig(ATCAIfaceCfg* cfg);
-
+#if defined(WOLFSSL_MICROCHIP_TA100)
+WOLFSSL_API int atmel_SetSharedDataConfig(ta_element_attributes_t* cfg);
```

Comment:
new public function should follow library naming scheme, i.e wolfCrypt_* or wolfSSL_*